inlong-manager/manager-web/src/main/resources/log4j2.xml

```diff
@@ -20,7 +20,7 @@
 <configuration status="WARN" monitorInterval="30" packages="org.apache.inlong.manager.web.config">
     <properties>
         <property name="basePath">logs</property>
-        <property name="log_pattern">%d{yyyy-MM-dd HH:mm:ss.SSS} -%5p [%5.30t] %-30.30C{1.}:%L - %mask %n</property>
+        <property name="log_pattern">%d{yyyy-MM-dd HH:mm:ss.SSS} -%5p [%5.30t] %-30.30C{1.}:%L - %enc{%mask}{CRLF} %n</property>
```

Comment:
Modifying the log4j2.xml file will affect all logs. For example, the output SQL cannot be copied and used directly, because symbols such as \r and \n will be output additionally, which is not conducive to the reading of logs.